### ISSUE TITLE
Accessibility - Dataverse - ARIA Roles must contain Children

### ISSUE THREAD

**Issue by tander08** [QualitativeDataRepository/dataverse#93]
### Homepage: https://data.qdr.syr.edu/
- [ ] Sort Menu 
![image](https://user-images.githubusercontent.com/89411909/134701082-5f0c03e1-3404-4544-807c-8242f2af58ab.png)

### (Sample Page): Data for: Qualitative Data Sharing: Participant Understanding, Motivation, and Consent: https://data.qdr.syr.edu/dataset.xhtml?persistentId=doi:10.5064/F6YYA3O3
- [ ] Cite Data Project Menu
![image](https://user-images.githubusercontent.com/89411909/135334766-e0bb8064-ce8e-4bf3-ad10-e6d85798e55b.png)
- [ ] Data Download Menu
![image](https://user-images.githubusercontent.com/89411909/135911195-a6fa7969-a1b4-4fbd-92dc-a9f46dcc6e4d.png)

### (Sample Page): M4 Eight Divided by One-Fourth Elementary Mathematics Task (Tree View)
https://data.qdr.syr.edu/dataset.xhtml?persistentId=doi:10.5064/F6BCQ6DM
![image](https://user-images.githubusercontent.com/89411909/140094762-6d1904a5-6d3d-4e84-94b1-f39a235d642b.png)

**Comment by qqmyers** [comment]
This issue is triggered because the dropdowns have role="menu" and the li items in the list do not have role="menuitem".
From web info such as https://web.archive.org/web/20190116170516/https://www.marcozehe.de/2018/09/22/wai-aria-menus-and-why-you-should-generally-avoid-using-them/, I get the sense that removing the role="menu" is the better fix here. Some, like the menu under 'New Project' open new pages and I think clearly don't match the intent of menu. Others (ones that let you select a download choice) might closer to what's expected, but I'm not sure that keeping the role="menu" really helps there either. If some/all should keep role="menu", let me know. Otherwise I'll probably scan the codebase and remove them all.

**Comment by adam3smith** [comment]
This looks fixed?

**Comment by qqmyers** [comment]
Yes - I did go ahead and remove all the menu roles which should have gotten rid of all of them.
